src/modules/txbuilder/RadixTransactionBuilder.ts

```diff
@@ -376,6 +381,104 @@ export default class RadixTransactionBuilder {
         return this
     }
 
+    /**
+     * Mint and transfer a multi-issuance token in one atom.
+     * 
+     * @param ownerAccount The owner of the token.
+     * @param tokenReference Token reference.
+     * @param decimalQuantity The amount to mint and transfer.
+     * @param from Sender account, needs to have RadixAccountTransferSystem.
+     * @param to Receiver account
+     * @param message Optional message.
+     */
+    public mintAndTransferTokens(
+        ownerAccount: RadixAccount,
+        tokenReference: string | RRI,
+        decimalQuantity: string | number | Decimal,
+        from: RadixAccount,
```

Comment:
This is superfluous, the sender will always be the same as the minter

Reply:
Resolved